apps/judicial-system/web/src/components/Table/CourtDate/CourtDate.tsx

```diff
@@ -0,0 +1,44 @@
+import { useIntl } from 'react-intl'
+import format from 'date-fns/format'
+import localeIS from 'date-fns/locale/is'
+import parseISO from 'date-fns/parseISO'
+
+import { Box, Text } from '@island.is/island-ui/core'
+import { capitalize } from '@island.is/judicial-system/formatters'
+import { tables } from '@island.is/judicial-system-web/messages'
+
+interface Props {
+  courtDate?: string | null
+  postponedIndefinitelyExplanation?: string | null
+}
+
+const CourtDate: React.FC<Props> = (props: Props) => {
```

Comment:
We are moving towards importing everything from React, instead of writing `React.whatever`. Please consider importing FC from react. Also, when you do `FC<Props>`, you don't have to explicitly say that the props are of type Props. This line could be written as 

`const CourtDate: FC<Props> = (props) => {`